src/Tribe/Aggregator/Cron.php

```diff
@@ -465,6 +442,42 @@ public function start_batch_pushing_records() {
 		}
 	}
 
+	/**
+	 * Get the first set of pending schedule records to be processed for batch pushing.
+	 *
+	 * @since TBD
+	 *
+	 * @return WP_Query The result of the Query.
+	 */
+	private function get_batch_pushing_records() {
+		return Records::instance()->query(
+			[
+				'post_status'    => Records::$status->pending,
+				'posts_per_page' => 250,
+				'order'          => 'ASC',
+				'meta_query'     => [
```

Comment:
:warning: Detected usage of meta_query, possible slow query.

<sub>hash: e9c17123cf540788e5fe1a3c0211db02</sub>